### PR TITLE
Added calibration macro for r3b-music

### PR DESCRIPTION
**Pull request by jose-luis-rs**
Added fit limits for r3b-music calibration

Solved compilation problems with Kelic-PR

### CODE REVIEW

File: r3b/run_digi.C [hunk 1]
```diff
@@ -4,8 +4,8 @@ void RemoveGeoManager();
 void run_digi()
 {
     // ----- Files ---------------------------------------------------------------
-    TString inFile = "sim.root";
-    TString parFile = "par.root";
+    TString inFile = "gen_sim.root";
+    TString parFile = "gen_par.root";
```
Comment (kresan) [inline r3b/run_digi.C]: This will make test run_digi failing. You need to skip this change

File: r3b/run_digi.C [hunk 2]
```diff
@@ -44,7 +44,7 @@ void run_digi()
     run->AddTask(sfi_digitizer);
 
     // dTOF
-    R3BTofdDigitizer* dtof_digitizer = new R3BTofdDigitizer();
+    R3BTofdDigitizer* tofd_digitizer = new R3BTofdDigitizer();
```
Comment (kresan) [inline r3b/run_digi.C]: Already fixed in dev

File: r3b/run_gen_sim.C [hunk 3]
```diff
@@ -2,8 +2,8 @@ void run_gen_sim()
 {
     TString transport = "TGeant3";
 
-    TString outFile = "gen_sim.root";
-    TString parFile = "gen_par.root";
+    TString outFile = "sim.root";
+    TString parFile = "par.root";
```
Comment (kresan) [inline r3b/run_gen_sim.C]: Will affect tests. Please skip it

**Comment by jose-luis-rs** [comment]
I think that it is ready!

**Pull request merged**